rct229/rulesets/ashrae9012019/ruleset_functions/get_min_oa_cfm_sch_zone.py

```diff
@@ -0,0 +1,72 @@
+from rct229.utils.assertions import assert_, getattr_
+from rct229.utils.jsonpath_utils import find_all, find_exactly_one_with_field_value
+
+
+def get_min_oa_cfm_sch_zone(rmi, zone_id, is_leap_year: bool = False):
+    """Each zone can have multiple terminal units sering it in the proposed RMR and each of these units could supply OA CFM. In order to obtain a zone level OA CFM schedule the OA CFM provided by each terminal unit needs to be aggregated for each hour of the year. This function receives an RMR (B, U, or P) and a zone ID and loops through each terminal unit associated with the zone to create an aggregated 8760 for OA CFM for the zone.
+
+    Parameters
+    ----------
+    rmi: json
+        The RMR in which the OA CFM schedule will be determined for the specific Zone ID.
+    zone_id:
+        The Zone ID in which the aggregated (across the terminal units serving the zone) hourly OA CFM schedule will be determined.
+    is_leap_year: bool, default: False
+        Whether the year is a leap year or not.
+
+    Returns
+    -------
+    list: An aggregated OA CFM hourly schedule for the zone (for each hour of the year, for each terminal unit, Terminal.minimum_outdoor_airflow is multiplied by Terminal.minimum_outdoor_airflow_multiplier_schedule, this product is summed across the terminal units for each hour of the year) .
+    """
+
+    min_OA_CFM_schedule_for_zone = None  # Returns None if the zone has no terminal or every terminal in the zone do not use minimum_outdoor_airflow_multiplier_schedule
```

Comment:
The context in the comment was suppose to be included in the function comment under the Returns section.

```md
Returns
     -------
     list: An aggregated OA CFM hourly schedule for the zone (for each hour of the year, for each terminal unit, Terminal.minimum_outdoor_airflow is multiplied by Terminal.minimum_outdoor_airflow_multiplier_schedule, this product is summed across the terminal units for each hour of the year) .
    None if the zone has no terminal or every terminal in the zone do not use minimum_outdoor_airflow_multiplier_schedule.
```

In this updated code, you actually initialize the `min_OA_CFM_schedule_for_zone` so you don't actually need to initialize it to None first.

Either way works for me. Since you modified the function to take in is_leap_year, I think it is OK to keep it this way but remove the line 22.

Reply:
Sounds great. Addressed.